src/pages/CookieClicker.vue

```diff
@@ -14,22 +15,29 @@ setInterval(() => {
 
 function cookieClick(){
     cookies.value += 1;
+    checkAchievements();
+}
+
+function checkAchievements() {
+    if (cookies.value === 10 && !achievements.value.includes("10 Clicks Milestone!")) {
+        achievements.value.push("10 Clicks Milestone!");
+        alert("Achievement Unlocked: 10 Clicks Milestone!");
```

Comment:
don't ever use alert in modern js or in any js for that matter. As long as alert is active page is frozen and that is bad... feel free to either throw a modal window up or pull in some library like SweetAlert